lib/charms/tls_certificates_interface/v4/tls_certificates.py

```diff
@@ -1025,7 +1025,7 @@ def _configure(self, _: EventBase):
         self._find_available_certificates()
         self._cleanup_certificate_requests()
 
-    def _mode_is_valid(self, mode) -> bool:
+    def _mode_is_valid(self, mode: Mode) -> bool:
```

Comment:
This file should be changed upstream. Not here.

Also, it shouldn't have been validated by pyright in the first place.

Reply:
I must have done this just because I came across it. It wasn't caught by pyright.